models/keyword.go

```diff
@@ -0,0 +1,59 @@
+package models
+
+import (
+	"errors"
+
+	"github.com/gutakk/go-google-scraper/db"
+	errorHandler "github.com/gutakk/go-google-scraper/helpers/error_handler"
+	"gorm.io/gorm"
+)
+
+const (
+	fileFormatError  = "File must be CSV format"
+	fileLengthError  = "CSV file must contain between 1 to 1000 keywords"
+	invalidDataError = "Invalid data"
+)
+
+type Keyword struct {
+	gorm.Model
+	Keyword string `gorm:"notNull;index"`
+}
+
+func ValidateFileType(fileType string) error {
+	if fileType != "text/csv" {
+		return errors.New(fileFormatError)
+	}
+	return nil
+}
+
+func ValidateCSVLength(row int) error {
+	if row <= 0 || row > 1000 {
+		return errors.New(fileLengthError)
+	}
+	return nil
+}
+
+func SaveKeywords(record [][]string) ([]Keyword, error) {
```

Comment:
What will be the CSV format be like? I am not sure why it needs to be an array of array of strings 🤔

Reply:
As I used `ReadAll()` function and the signature look like this
```go
func (r *Reader) ReadAll() (records [][]string, err error) {...}
```

Reply:
We can discuss about this in this [comment](https://github.com/gutakk/go-google-scraper/pull/25#discussion_r532476260)